transforms/CANFD.yaml

```diff
@@ -0,0 +1,150 @@
+transforms:
```

Comment:
Just for future reference, can you put a comment saying what chip this was taken from (e.g. G350x)